### ISSUE TITLE
Copy images on paste to other page instead of linking

### ISSUE THREAD

**Issue by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#45]
Pasting an image from one page to another results in a link instead of a copy, this is not what users expect. Even more for generated images. E.g. copying an equation and then editing it also changes the original.

Copy-paste should ask whether to copy or link a) images and b) other attachments linked in the text, but only for  images that are direct attachments. Images that are e.g. in the document root will always be linked.

**Comment by jaap-karssenberg** [comment]
https://bugs.launchpad.net/zim/+bug/330320

**Comment by ziggystar** [comment]
Maybe related:  #418

**Comment by LeoUfimtsev** [comment]
I run into this issue when I copy & paste text that contains an image from one page onto another. The pasted text has a broken image.

Would really *love* to see this fixed.

**Comment by Osndok** [comment]
Also relates to (and limits the utility of) the "Move selected text..." context menu, which (IIRC) leaves broken image links ATM.

**Comment by FlorianWanders** [comment]
Could this issue be solved by having a global attachments folder where the notebook.zim file is located?  (or possibly even one folder further up so that the attachments folder does not show in the index, although maybe there is also a way of hiding it in the index)

**Comment by xbingnan** [comment]
Any updates on this issue? still not fixed...

**Comment by jaap-karssenberg** [comment]
If there was an update, you would read it in the issue. Feel free to start
a patch for this.

**Comment by Xunius** [comment]
I got the same issue as described by @xbingnan: copying a block of texts + images and pasting into another note results in broken image links. Right clicking a single image and copying-pasting does produce the image in the target page, however, the image is not saved to a subfolder named after the target page. I'm not sure whether this will lead to any future issues.

I found this problem makes re-structuring of the notes pretty difficult, I have to manually re-insert all the related images to relocate a section of note to a different page. Hope this could be fixed soon. I agree with LeoUfimtsev that this is the only issue I'm having with zim, otherwise it is great.

btw I'm using 0.73.2
